go_modules/lib/dependabot/go_modules/file_fetcher.rb

```diff
@@ -15,6 +15,8 @@ def self.required_files_message
       end
 
       def package_manager_version
+        return nil unless go_mod
```

Comment:
Verified that we're handling `nil` in [the FileFetcherCommand](https://github.com/dependabot/dependabot-core/blob/5e1be21dbffd749203af347478e6d5a15a180181/updater/lib/dependabot/file_fetcher_command.rb#L20)